index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Kajal Pramanik</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  
+  <header>
+    <nav>
+      <div class="nav-bar">
+        <a href="#" class="logo">Kajal</a>
+        <button class="hambug-menu">
+         <a href="#"><img src="./images/menu.svg" alt="menu"></a>
+        </button>
```

Comment:
[OPTIONAL] 👍🏽 

- I believe it will be better to use either a button alone for your hamburger menu or an `<a>` tag, instead of using both. We should try to make our code base as DRY as possible. Why don't we pick one and delete the other since they are both referencing a clickable link 🌸